packages/node_modules/@node-red/nodes/examples/common/debug/1 example.json

```diff
@@ -0,0 +1 @@
sole, or status below the node.  It can output payload, complete object, or result of JSONata expression.","x":140,"y":60,"wires":[]},{"id":"87bd706a.aec93","type":"comment","z":"3ae4b3d9.1f77bc","name":"Simple output to debug sidebar","info":"","x":210,"y":120,"wires":[]},{"id":"8035b07f.7547e","type":"inject","z":"3ae4b3d9.1f77bc","name":"","topic":"","payload":"Hello, World!","payloadType":"str","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":210,"y":160,"wires":[["20d1344f.931e3c"]]},{"id":"20d1344f.931e3c","type":"debug","z":"3ae4b3d9.1f77bc","name":"","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"false","x":450,"y":160,"wires":[]},{"id":"e4b97778.b4b4b8","type":"comment","z":"3ae4b3d9.1f77bc","name":"Output to debug sidebar & console","info":"","x":220,"y":220,"wires":[]},{"id":"41f7b1de.6de52","type":"inject","z":"3ae4b3d9.1f77bc","name":"","topic":"","payload":"Hello, World!","payloadType":"str","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":210,"y":260,"wires":[["4aa37fe0.91341"]]},{"id":"4aa37fe0.91341","type":"debug","z":"3ae4b3d9.1f77bc","name":"","active":true,"tosidebar":true,"console":true,"tostatus":false,"complete":"payload","targetType":"msg","x":460,"y":260,"wires":[]},{"id":"268b2307.053f8c","type":"comment","z":"3ae4b3d9.1f77bc","name":"Output to debug sidebar & status","info":"","x":210,"y":320,"wires":[]},{"id":"a7bf2e39.6738f","type":"inject","z":"3ae4b3d9.1f77bc","name":"","topic":"","payload":"Hello, World!","payloadType":"str","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":210,"y":360,"wires":[["9fd6d23e.25689"]]},{"id":"9fd6d23e.25689","type":"debug","z":"3ae4b3d9.1f77bc","name":"","active":true,"tosidebar":true,"console":false,"tostatus":true,"complete":"payload","targetType":"msg","x":450,"y":360,"wires":[]},{"id":"2048c1f6.e72bee","type":"comment","z":"3ae4b3d9.1f77bc","name":"Output complete object","info":"","x":180,"y":420,"wires":[]},{"id":"fe9b300e.bdb96","type":"inject","z":"3ae4b3d9.1f77bc","name":"","topic":"Sample","payload":"Hello, World!","payloadType":"str","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":240,"y":460,"wires":[["5b7d830c.00bdbc"]]},{"id":"5b7d830c.00bdbc","type":"debug","z":"3ae4b3d9.1f77bc","name":"","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"true","targetType":"full","x":430,"y":460,"wires":[]},{"id":"d923a725.4ff058","type":"comment","z":"3ae4b3d9.1f77bc","name":"Output complete object","info":"","x":180,"y":520,"wires":[]},{"id":"e97d46cb.65fdb8","type":"inject","z":"3ae4b3d9.1f77bc","name":"","topic":"Sample","payload":"Hello, World!","payloadType":"str","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":240,"y":560,"wires":[["9441873.2c7ef78"]]},{"id":"9441873.2c7ef78","type":"debug","z":"3ae4b3d9.1f77bc","name":"","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"\"[\" & topic & \"] \" & payload","targetType":"jsonata","x":440,"y":560,"wires":[]}]
```

Comment:
on final example - the comment box should probably have a more descriptive title - "Formatting debug using JSONata" or something like that ?

Reply:
changed comment title according to your suggestion.